### PR TITLE
tweak manifest

### PR DESCRIPTION
**Pull request by copejon**
removed redundant RUN mkdir from Dockerfile (k8s will create per mountPath); reworked key names

### CODE REVIEW

File: build/importer/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,3 @@
 FROM busybox
 COPY ./importer /importer
-RUN mkdir /image
```
Comment (copejon) [inline build/importer/Dockerfile]: forgot k8s will make whatever dir the mountPath is

File: manifests/importer-pod-config.yaml [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 data:
-  importer_url: ""  #intentionally blank
-  importer_endpoint: ""
-  # importer_object_path: <bucket-name>/<object-name>
-  importer_object_path: ""
-  importer_dest_path: "/image"
+  url: ""               # mutually exclusive w/ endpoint
+  endpoint: ""
+  objectPath: ""        # expects: <bucket-name>/<object-name>
+  writeToPath: "/image" # MUST match importer pod spec.containers[0].volumeMounts[0].mountPath
```
Comment (jeffvance) [inline manifests/importer-pod-config.yaml]: says newline is missing...

File: manifests/importer-pod.yaml [hunk 3]
```diff
@@ -7,45 +7,44 @@ spec:
   restartPolicy: Never
   containers:
   - name: importer
-    image: docker.io/jcoperh/importer:jcope-latest 
+    image: docker.io/jcoperh/importer:latest
     imagePullPolicy: Always
```
Comment (jeffvance) [inline manifests/importer-pod.yaml]: not IfNotPresent? Or is it never present?

File: manifests/importer-pod.yaml [hunk 4]
```diff
@@ -7,45 +7,44 @@ spec:
...
+          name: importer-config
+          key: endpoint
     - name: IMPORTER_OBJECT_PATH
       valueFrom:
         configMapKeyRef:
-          name: importer-pod-config
-          key: importer_object_path
+          name: importer-config
+          key: objectPath
     - name: IMPORTER_DEST_PATH
```
Comment (jeffvance) [inline manifests/importer-pod.yaml]: we're keeping `IMPORTER_DEST_PATH` as the var name but it won't match the config key name now.

**Comment by jeffvance** [comment]
@copejon also can you `rm` the 2 manifests _importer-controller*_?

File: manifests/importer-pod.yaml (hunk 3 above)
Comment (copejon) [reply manifests/importer-pod.yaml]: we always want the latest, at least for now, because we're testing changes

**Pull request merged**